### ISSUE TITLE
[Refactor] AlertBanner - replace Button + Icon with IconButton

### ISSUE THREAD

**Issue by talkor** [mondaycom/vibe#2084]
In `<AlertBanner>`, currently a `<Button>` and `<Icon>` are used for the close button. Should be replaced with `<IconButton>` while preserving all abilities and a11y.

Monday ticket: https://monday.monday.com/boards/3532714909/pulses/5414954008

**Comment by YossiSaadi** [comment]
Resolved at #2120
